src/main/webapp/projectContainer/ConsentGroups.js

```diff
@@ -77,82 +171,140 @@ export const ConsentGroups = hh(class ConsentGroups extends Component {
+    e.stopPropagation();
+    this.setState(prev => {
+      prev.action = 'approve';
+      prev.showConfirmationModal = true;
+      prev.actionConsentKey = consentKey;
+      return prev;
     });
+  };
 
-    $(".confirmationModal").on('click', function () {
-      rThis.setState(prev => {
-        prev.showConfirmationModal = true;
-        prev.action = $(this).data("handler");
-        prev.issueKey = $(this).data("issue");
-        prev.consentKey = $(this).data("consent");
-        return prev;
-      });
+  reject = (e, consentKey) => {
+    e.stopPropagation();
+    this.setState(prev => {
+      prev.action = 'reject';
+      prev.showConfirmationModal = true;
+      prev.actionConsentKey = consentKey;
+      return prev;
     });
-    $(".request-clarification").on('click', function () {
-      rThis.setState(prev => {
-        prev.showRequestClarification = true;
-        prev.issueKey = $(this).data("issue");
-        prev.consentKey = $(this).data("consent");
-        return prev;
-      });
+  };
+
+  unlink = (e, consentKey) => {
+    e.stopPropagation();
+    this.setState(prev => {
+      prev.action = 'unlink';
+      prev.showConfirmationModal = true;
+      prev.actionConsentKey = consentKey;
+      return prev;
+    });
+  };
+
+  requestClarification = (e, consentKey) => {
+    e.stopPropagation();
+    this.setState(prev => {
+      prev.showRequestClarification = true;
+      prev.actionConsentKey = consentKey;
+      return prev;
     });
-    // Display for 8 seconds a message indicating the submission of a new consent group. 
-    // This is temporary until this page is moved to react.
-    // https://broadinstitute.atlassian.net/browse/BTRX-628
-    var url = new URLSearchParams(window.location.search);
-    if (url.get('tab') === 'consent-groups' && url.has('new')) {
-      $('#alert').fadeIn('slow', function () {
-        $('#alert').delay(15000).fadeOut();
-        history.pushState({}, null, window.location.href.split('&')[0]);
+  };
+
+  parseCollapsibleElementsData = (consents) => {
+    let parsedArray = [];
+    if (!isEmpty(consents)) {
+      parsedArray = consents.map(consent => {
+        return {
+          consent : consent,
+          search : false,
+          remoteProp : false,
+          data: consent.attachments,
+          columns: columns(this),
+          keyField: 'id',
+          defaultSorted: defaultSorted,
+          fileName: '_',
+          pagination: false,
+          showExportButtons : false,
+          showSearchBar : false,
+          customHandlers : {
+            approveHandler: this.approve,
+            rejectHandler: this.reject,
+            unlinkHandler: this.unlink,
+            requestClarificationHandler: this.requestClarification
+          }
+        }
       });
     }
+    return parsedArray;
+  };
+
+  redirect = (action) => {
+    const path = action === 'new' ? '/consent-group/new?projectKey='+ component.projectKey : '/consent-group/use-existing';
```

Comment:
```suggestion
    const path = action === 'new' ? UrlConstants.updateConsentGroupUrl +?projectKey='+ component.projectKey : UrlConstants.consentGroupExistingUrl;
```
This last one should be added to UrlConstants